hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/hip551/HollowAccountsAndKeysBatchTest.java

```diff
@@ -142,7 +144,9 @@ public Stream<DynamicTest> hollowSendFundsAfterSigRequireUpdateWithoutReceiverSi
                 getAliasedAccountInfo(alias).isHollow()));
     }
 
-    @HapiTest
+    @LeakyHapiTest(
+            requirement = {THROTTLE_OVERRIDES},
+            throttles = "testSystemFiles/mainnet-throttles.json")
```

Comment:
Why are we overriding the throttles here? This test is focused on fees, so it's probably a symptom, not a cause